### ISSUE TITLE
Audio drops when on Android Auto

### ISSUE THREAD

**Issue by wings2004** [futo-org/grayjay-android#913]
While listening in the background or watching, the audio cuts out after about 2 minutes. Starting another audio source such as YouTube allows me to get the audio working again but after a couple more minutes it cuts out again. The video is still playing and there is no indication why the audio isn't still working.

Edit: this only appears to happen on Android Auto and has not happened on normal Bluetooth headphone connections.

**Comment by wings2004** [comment]
Wanted to add a work around I've found so far. Assuming the audio is working in general, I have been able to get Grayjay to behave normally by swiping the app closed fully and then reopen. 

The app has a tendency to have a slower load on the initially but this could be a result of being a fully startup vs a resume from background and this also being on cellular data when reloading the app.

**Comment by beemandev** [comment]
Yes, this has been annoying me too for months. My only workaround is to use close all apps, re-run GrayJay, history, resume. No other apps have this weird issue. Very easy to reproduce, just run it, turn off car, turn car back on, play = plays but no audio. Pressing play on another app, you will hear both, stop second app, GrayJay goes silent after a few seconds but continues playing. It's as if it doesn't connect to correct audio output, dunno where it does send audio? Video fine though.
